### ISSUE TITLE
about pretrain acc

### ISSUE THREAD

**Issue by sunny-zzw** [noahzn/Lite-Mono#146]
Thank you for this work and i want to consult about the acc1 and acc5 of the pretrain result on the ImageNet

**Comment by noahzn** [comment]
Hi, I don't remember the exact accuracy for pre-training. It should be around 73%.
